data/Localization/german_(germany)/global.ini

```diff
@@ -87,7 +87,7 @@ Alerts_BeingArrested=Bleiben Sie stehen, sie werden nun verhaftet
 Alerts_BeingScanned=bleiben sie stehen, Scan läuft
 AmbassadorFlights_Allies=Silver Leaf Society
 AmbassadorFlights_RepUI_Area=UEE
-AmbassadorFlights_RepUI_Description=Ambassador Flights hat sich einen Ruf als bevorzugter Shuttle-Service von Führungskräften und VIPs im ganzen Empire erworben. Zwar verfügt Ambassador Flights über eine eigene Flotte von Premium-Schiffen, mit denen sie ihre wohlhabende Kundschaft befördern, doch um ihren Service weiter auszubauen, nehmen sie auch Privatpiloten unter Vertrag, die ihren hohen Ansprüchen an Hygiene, Komfort und Privatsphäre gerecht werden.
+AmbassadorFlights_RepUI_Description=Ambassador Flights hat sich einen Ruf als bevorzugter Shuttle-Service von Führungskräften und VIPs im ganzen Empire erworben. Zwar verfügt Ambassador Flights über eine eigene Flotte von Premium-Schiffen, mit denen sie ihre wohlhabende Kundschaft befördern, doch um ihren Service weiter auszubauen, nehmen sie auch Privatpiloten unter Auftrag, die ihren hohen Ansprüchen an Hygiene, Komfort und Privatsphäre gerecht werden.
```

Comment:
In dem Satz am Ende wäre Vertrag vielleicht doch das passendere Wort. 
Außerdem am Ende des ersten Satzes das Wort "erworben" mit "verdient" ersetzen? Klingt besser, finde ich.